ui/common/src/main/java/com/appcoins/wallet/ui/common/theme/WalletColors.kt

```diff
@@ -20,4 +20,6 @@ object WalletColors {
   val styleguide_orange = Color(0xFFf86d00)
   val styleguide_vip_yellow = Color(0xFFEFA226)
   val styleguide_vip_yellow_transparent_40 = Color(0x66EFA226)
+  val styleguide_skeleton_loading = Color(0xFF2E2D3C)
```

Comment:
I think this color is correct, but check the 3 colors used in the topup skeletons/shimmer, so that they have the same values:
-  styleguide_topup_skeleton
- styleguide_topup_skeleton_shimmer
- styleguide_topup_skeleton_secondary

Reply:
Luis told us to leave the current color palettes that are correct with the design